tests/fafnir_tests/mod.rs

```diff
@@ -249,4 +268,33 @@ pub fn main_test(mut es_wrapper: ElasticSearchWrapper, pg_wrapper: PostgresWrapp
     assert!(&le_nomade.is_poi());
     let le_nomade = &le_nomade.poi().unwrap();
     assert_eq!(&le_nomade.id, "osm:way:42"); // the id in the database is '-42', so it's a way
+
+    // Test that the airport 'Isla Cristina Agricultural Airstrip' has been imported in the elastic wrapper
+    let airport_cristina: Vec<mimir::Place> = es_wrapper
+        .search_and_filter("Isla Cristina", |_| true)
+        .collect();
+    assert_eq!(&airport_cristina.len(), &1);
+    assert!(&airport_cristina[0].is_poi());
+
+    // Test the airport id
+    let airport = &airport_cristina[0].poi().unwrap();
+    assert_eq!(&airport.id, "osm:node:4505823836");
+
+    // Test the airport coord
+    let airport_coord = &airport.coord;
+    assert_eq!(&airport_coord.lat(), &37.24221674256237);
```

Comment:
why the ref ?

double equals should be wrapped with approx

Reply:
done